src/navigation/components/NavigationItem/components/NavigationDropdown.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import {NavigationItemProps} from '../NavigationItem';
+import {Content} from './Content';
+import {ToggleArrow} from '../../../../components';
+
+import {block} from '../../../../utils';
+import {getMediaImage} from '../../../../components/Media/Image/utils';
+
+import {DropdownItemData} from '../../../../models';
+
+import '../NavigationItem.scss';
+
+const b = block('navigation-item');
+
+const TOGGLE_ARROW_SIZE = 12;
+
+type NavigationDropdownProps = NavigationItemProps & DropdownItemData;
+
+export const NavigationDropdown: React.FC<NavigationDropdownProps> = ({
+    text,
+    icon,
+    isOpened,
+    ...props
+}) => {
+    const iconData = icon && getMediaImage(icon);
+
+    return (
+        <span {...props}>
```

Comment:
Why it is span?

Reply:
it's not my feature, it's historical moment, maybe @gorgeousvlad will give the answer